jupyterlab_bigquery/src/components/query_history/query_history_panel.tsx

```diff
@@ -63,63 +114,177 @@ interface Props {
 interface State {
   hasLoaded: boolean;
   isLoading: boolean;
-  jobIds: JobIdsObject;
+  jobIds: string[];
   jobs: JobsObject;
   openJob: string;
 }
 
-const QueryDetails = props => {
-  const job = props.details;
+const ErrorBox = (props: { errorMsg: string }) => {
+  return (
+    <Paper
+      style={{
+        display: 'flex',
+        alignItems: 'stretch',
+        marginBottom: '12px',
+      }}
+      elevation={1}
+      variant="outlined"
+    >
+      <div style={{ width: '6px', backgroundColor: '#e60000' }} />
+      <div style={{ padding: '8px', display: 'flex', alignItems: 'center' }}>
+        <Error
+          fontSize="default"
+          htmlColor="rgb(230, 0, 0)"
+          className={localStyles.icon}
+        />
+        {props.errorMsg}
+      </div>
+    </Paper>
+  );
+};
+
```

Comment:
This is the box at the top of the expanded query details that shows what error caused the query to fail. We may pull this out as a shared component later since it seems the empty schema and empty table display uses a similar format.